@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -338,37 +333,36 @@
 }
 
 .navds-combobox__list-item--selected {
-  background-color: var(--ac-combobox-list-item-selected-bg, var(--a-surface-selected));
+  background-color: var(--ax-bg-accent-moderate-pressedA);
 }
 
 .navds-combobox__list-item--selected p {
-  font-weight: var(--a-font-weight-bold);
+  font-weight: var(--ax-font-weight-bold);
 }
 
 .navds-combobox__list-item--selected.navds-combobox__list-item--focus,
 .navds-combobox__list--with-hover .navds-combobox__list-item--selected:hover {
-  background-color: var(--ac-combobox-list-item-selected-hover-bg, var(--a-surface-action-subtle-hover));
-  border-left: 4px solid var(--ac-combobox-list-item-selected-hover-border-left, var(--a-border-focus));
+  background-color: var(--ax-bg-accent-moderateA);
 }
 
 .navds-combobox__list-item--new-option {
-  border-bottom: 1px solid var(--a-border-divider);
-  background: var(--a-surface-neutral-subtle);
+  border-bottom: 1px solid var(--ax-border-accent-subtleA);
+  background: var(--ax-bg-accent-moderateA);
   cursor: pointer;
   justify-content: flex-start;
   gap: 0.25rem;
+  margin: 0;
+  border-radius: 0;
+  width: calc(100% + var(--ax-spacing-4));
```

Comment:
Is there a need for different handling of overflow now?

Reply:
this was to make the "add new" option match the new design, without these lines it looks like this 
![image](https://github.com/user-attachments/assets/1f77241a-f488-403a-a3e1-84d82fd55aea)

the width calculation is probably something that'll change once I figure out the scroll situation... it shouldn't have to be this hard 😅 (to test it during development, i also found that commenting out these lines helped https://github.com/navikt/aksel/blob/main/%40navikt/core/react/src/form/combobox/ComboboxWrapper.tsx#L42-L44)